src/style.css

```
@@ -0,0 +1,79 @@
+body {
+  margin: 0;
+}
+
+.blocks {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  text-align: center;
+  flex-basis: 33.3%;

```

```suggestion
  flex-basis: 33.33%;
```
Указывай значения с точностью до сотых, так элементы будут растягиваться правильно, не оставляя белых полос